project/cmake/modules/FindVAAPI.cmake

```diff
@@ -39,27 +41,33 @@ endif()
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(VAAPI
-                                  REQUIRED_VARS VAAPI_libva_LIBRARY VAAPI_libva-x11_LIBRARY VAAPI_INCLUDE_DIR
+                                  REQUIRED_VARS VAAPI_libva_LIBRARY VAAPI_libva-x11_LIBRARY VAAPI_libva-drm_LIBRARY VAAPI_INCLUDE_DIR
                                   VERSION_VAR VAAPI_VERSION_STRING)
 
 if(VAAPI_FOUND)
   set(VAAPI_INCLUDE_DIRS ${VAAPI_INCLUDE_DIR})
-  set(VAAPI_LIBRARIES ${VAAPI_libva_LIBRARY} ${VAAPI_libva-x11_LIBRARY})
+  set(VAAPI_LIBRARIES ${VAAPI_libva_LIBRARY} ${VAAPI_libva-x11_LIBRARY} ${VAAPI_libva-drm_LIBRARY})
   set(VAAPI_DEFINITIONS -DHAVE_LIBVA=1)
 
   if(NOT TARGET VAAPI::VAAPI_X11)
     add_library(VAAPI::VAAPI_X11 UNKNOWN IMPORTED)
     set_target_properties(VAAPI::VAAPI_X11 PROPERTIES
                                            IMPORTED_LOCATION "${VAAPI_libva-x11_LIBRARY}")
   endif()
+  if (NOT TARGET VAAPI::VAAPI_DRM)
+    add_library(VAAPI::VAAPI_DRM UNKNOWN IMPORTED)
+    set_target_properties(VAAPI::VAAPI_DRM PROPERTIES
+                                           IMPORTED_LOCATION "${VAAPI_libva-drm_LIBRARY}")
+  endif()
   if(NOT TARGET VAAPI::VAAPI)
     add_library(VAAPI::VAAPI UNKNOWN IMPORTED)
     set_target_properties(VAAPI::VAAPI PROPERTIES
                                        IMPORTED_LOCATION "${VAAPI_libva_LIBRARY}"
                                        INTERFACE_INCLUDE_DIRECTORIES "${VAAPI_INCLUDE_DIR}"
                                        INTERFACE_COMPILE_DEFINITIONS HAVE_LIBVA=1
-                                       INTERFACE_LINK_LIBRARIES VAAPI::VAAPI_X11)
+                                       INTERFACE_LINK_LIBRARIES VAAPI::VAAPI_X11
+                                       INTERFACE_LINK_LIBRARIES VAAPI::VAAPI_DRM)
```

Comment:
Docs state:
>This property contains the list of transitive link dependencies.

so this should work:
INTERFACE_LINK_LIBRARIES "VAAPI::VAAPI_X11 VAAPI::VAAPI_DRM"